src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -258,17 +288,29 @@ public enum MysteriousTreasures implements LogicCardInfo {
           move "Tail Influence", {
             text "30 damage. Your opponent flips a coin until he or she gets heads. For each tails, remove an Energy card attached to the Defending Pokémon and put it on the bottom of your opponent’s deck."
             energyCost C, C
-            attackRequirement {}
+            attackRequirement {
+              assert defending.cards.any{it.cards.filterByType(ENERGY)} : "The defending Pokémon has no Energy cards attached to them"
```

Comment:
This attack does damage already, is it correct to disallow it just because the opponent has not energy attached?

Reply:
Yeah, I recall adding the damage at last since I missed it. Coded it thinking it only discarded thanks!